bugbug/models/backout.py

```diff
@@ -9,13 +9,13 @@
 from sklearn.feature_extraction import DictVectorizer
 from sklearn.pipeline import Pipeline
 
-from bugbug import commit_features, feature_cleanup, repository
+from bugbug import bug_features, commit_features, feature_cleanup, repository
 from bugbug.model import CommitModel
 
 
 class BackoutModel(CommitModel):
     def __init__(self, lemmatization=False):
```

Comment:
I'd add an option here (`bug_data`), so the user can decide whether to use bug features or not.

Reply:
You should also actually only use the bug features when "bug_data" is True.

Reply:
```Python
if bug_data:
    feature_extractors += [
            bug_features.product(),
            bug_features.component(),
            ...
    ]
```

Reply:
Aren't these lines already taking care of that?
https://github.com/mozilla/bugbug/blob/b456bd3b2d00aad453b42cc8761fc04fbf0e3866/bugbug/commit_features.py#L178-L181

Reply:
Yes, but that check is not specifically for this, so we can't rely on it without making the code harder to maintain.
If we rely on checks that are there for other reasons, we introduce hidden dependencies in the code, which make it harder to modify later (what if somebody removes the "if not commit[..." check because they don't see that it is useful for the "backout" model?).